.github/workflows/reuse.reusable.yml

```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: 2022 Free Software Foundation Europe e.V. <https://fsfe.org>
+# SPDX-License-Identifier: CC0-1.0
+
+name: REUSE Compliance Check (reusable)
+
+on:
+  workflow_call:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: REUSE Compliance Check
+      uses: fsfe/reuse-action@v1
```

Comment:
Missing tailing newlines

Reply:
Done :) I also fixed indention for the lists.